modules/soe_paragraph_cta_list/config/install/field.field.paragraph.cta_list.su_cta_list_deck.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 2c1953b8-c160-4f04-a74b-612a6470886f
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.su_cta_list_deck
+    - paragraphs.paragraphs_type.cta_list
+  module:
+    - allowed_formats
+    - text
+third_party_settings:
+  allowed_formats:
+    stanford_html: stanford_html
+    stanford_minimal_html: stanford_minimal_html
```

Comment:
These come from https://github.com/SU-SWS/stanford_text_editor. Be sure to add to `composer.json` and `info.yml` dependencies.

Reply:
corrected.